### ISSUE TITLE
brave://settings/?search=Speedreader link doesn't auto-scroll to highlighted Speedreader option

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#19218]
## Description 

(I wonder if this is an upstream Chromium bug - I quickly tried to find a corollary there before filing.)

brave://settings/?search=Speedreader link doesn't auto-scroll to highlighted Speedreader option

## Steps to Reproduce

   1. new profile
   2. launch Brave
   3. load an Speedreader-capable article, such as `https://www.wired.com/story/ea-sports-might-lose-fifa-and-thats-not-so-bad/`
   4. click on the Speedreader icon in the URL bar
   5. in the popup, click on `Turn on Speedreader`
   6. click again on the Speedreader icon in the URL bar
   7. click on `Settings`
   8. you're taken to `brave://settings/?search=Speedreader`
   9. without scrolling or changing the window size, look for the Speedreader entry

The above steps are a good real-world use-case; however, it's just as easy to load `brave://settings/?search=Speedreader` to reproduce this, for me.

## Actual result:

On my MacBook Pro, using `Default for display` resolution, the Speedreader entry can't be seen without scrolling

<img width="1312" alt="Screen Shot 2021-11-03 at 1 32 20 PM" src="https://user-images.githubusercontent.com/387249/140189307-8365de57-1af5-4b67-94d1-6cd1eabc35c9.png">

## Expected result:

Should auto-scroll the viewport to show the highlighted setting, like so

<img width="1312" alt="Screen Shot 2021-11-03 at 1 33 18 PM" src="https://user-images.githubusercontent.com/387249/140189383-8e28b98a-6c6b-47e9-9895-239d32ee7c47.png">

## Reproduces how often: 

100% (for me)

## Brave version (brave://version info)

brave://settings/?search=Speedreader link doesn't auto-scroll to Speedreader option

Putting in `feature/speedreader` for now for triage, but do punt if there's a better component.

**Comment by stephendonner** [comment]
Yeah, this is more general of an issue: `brave://settings/?search=rewards` is affected as well.

example | example
---------|---------
<img width="1312" alt="Screen Shot 2021-11-03 at 1 47 55 PM" src="https://user-images.githubusercontent.com/387249/140190467-5c52827e-76f0-4246-a6e5-24175851a511.png"> | <img width="1312" alt="Screen Shot 2021-11-03 at 1 47 59 PM" src="https://user-images.githubusercontent.com/387249/140190476-8631bf27-a03b-4d00-8eed-71b354426a90.png">

@mkarolin / @petemill any idea if this can be fixed from our side, or know if there is/we need an upstream bug?

**Comment by rebron** [comment]
This one looks stale as Speedreader setting was changed. Not able to reproduce step 8. 
![Screenshot 2024-06-06 at 3 37 11 PM](https://github.com/brave/brave-browser/assets/5951041/d9ad614f-4654-425f-bce4-0b4f553eb6bd)

**Comment by rebron** [comment]
Closing as stale.
